src/dispatch/monitor/flows.py

```diff
@@ -0,0 +1,14 @@
+from dispatch.plugin import service as plugin_service
+
+
+def send_monitor_notification(project_id, conversation_id, message_template, db_session, **kwargs):
+    """Sends a monitor notification."""
+    notification_text = "Incident Notification"
+    notification_type = "incident-notification"
+
+    plugin = plugin_service.get_active_instance(
+        db_session=db_session, plugin_type="conversation", project_id=project_id
+    )
+    plugin.instance.send(
```

Comment:
We can... but I think I would want to know if this ever was the case as an explicit error. Because these monitors are only available via the conversation plugin, it would be unexpected to send this notification (as you wouldn't have been able to create it without the plugin enabled).